MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -5,6 +5,8 @@
 import UIKit
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching {
+	@IBOutlet private(set) weak var errorView: ErrorView!
```

Comment:
Same here as for the MVP solution:
```suggestion
	@IBOutlet public private(set) weak var errorView: ErrorView!
```